services/specialist-publisher/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: '3.7'
+
+x-specialist-publisher: &specialist-publisher
+  build:
+    context: .
+    dockerfile: services/specialist-publisher/Dockerfile
+  image: specialist-publisher
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+    - home:/home/build
+  working_dir: /govuk/specialist-publisher
+
+services:
+  specialist-publisher-default:
+    <<: *specialist-publisher
+    privileged: true
+    depends_on:
+      - mongo
+      - redis
+    environment:
+      MONGODB_URI: "mongodb://mongo/specialist-publisher"
+      TEST_MONGODB_URI: "mongodb://mongo/specialist-publisher-test"
+      REDIS_URL: redis://redis
+
+  specialist-publisher-backend: &specialist-publisher-backend
+    <<: *specialist-publisher
+    depends_on:
+      - mongo
+      - redis
+      - nginx-proxy
+      - publishing-api-backend
+      - asset-manager-backend
+      - email-alert-api-backend
+    environment:
+      MONGODB_URI: "mongodb://mongo/specialist-publisher"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: specialist-publisher.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    command: bin/rails s -P /tmp/rails.pid
+
+  specialist-publisher-e2e:
+    <<: *specialist-publisher-backend
+    depends_on:
+      - mongo
+      - redis
+      - specialist-publisher-worker
+      - publishing-api-e2e
+      - asset-manager-e2e
+      - email-alert-api-e2e
```

Comment:
There's no `email-alert-api-e2e`, but there is an `email-alert-api-backend`: https://github.com/alphagov/govuk-docker/pull/8/files

Reply:
I'm to leave making any changes until PR #8 has been merged in.

Reply:
Updated and fixed up.